kuama_core/lib/kuama_core.dart

```diff
@@ -2,8 +2,6 @@
 
 library kuama_core;
 
-export 'package:tuple/tuple.dart';
-
```

Comment:
Also remove tuple package from pubspec.yaml and add the modification to changelog